packages/fluentui/docs/src/examples/components/Form/Usage/FormExampleErrorAndSatisfactory.shorthand.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
```

Comment:
please add children api example as well

Reply:
Oh, ok, I thought was supposed to be a separated task. Will added it here.

Reply:
Added.